tests/unit_test/action/kremote_action_test.py

```
@@ -0,0 +1,103 @@
+    subpath = "path/to/resource"
+    method = "post"
+    success_response_data = {"key": "value"}
+
+    with aioresponses() as mock:
+        mock.post(f"http://test.com/{subpath}", status=500)
+        mock.post(f"http://test.com/{subpath}", status=500)
+        mock.post(f"http://test.com/{subpath}", status=500)
+        mock.post(f"http://test.com/{subpath}", status=200, payload=success_response_data)
+
+        with pytest.raises(ClientResponseError) as exc_info:
+            result = await KRemoteAction.multi_try_rasa_request(
```

Ruff

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)